common/src/main/java/dev/architectury/platform/Platform.java

```diff
@@ -51,12 +51,16 @@ private static void updateLoaderCache() {
             return;
         }
         
-        switch (ArchitecturyTarget.getCurrentTarget()) {
+        switch (getLoader()) {
             case "fabric" -> simpleLoaderCache = 0;
             case "forge" -> simpleLoaderCache = 1;
         }
     }
     
+    public static String getLoader() {
```

Comment:
We shouldn't expose this, ArchitecturyTarget.getCurrentTarget() is the compiled target, so on quilt it is still fabric because this is compiled for fabric. This has quite specific implications.